.github/workflows/unity.yml

```diff
@@ -0,0 +1,249 @@
+name: Draco Unity library CI
+
+on:
+  # Trigger the workflow on pull request,
+  # but only for the main branch
+  pull_request:
+    branches:
+      - main
+  # Also trigger on release created events
+  release:
+    types:
+      - created
+  workflow_dispatch:
+
+
+jobs:
+
+  mac:
+
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: latest-stable
+
+    - uses: seanmiddleditch/gha-setup-ninja@master
+
+    - name: configure_mac
+      # ARCHS_STANDARD explicitely enables Apple silicon on Xcode 12.2
+      run: cmake . -G Ninja -B build_mac -DCMAKE_BUILD_TYPE=MinSizeRel -DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF
+    - name: build_mac_dracodec_unity
+      run: cmake --build build_mac --config MinSizeRel --target dracodec_unity
+    - name: build_mac_dracoenc_unity
+      run: cmake --build build_mac --config MinSizeRel --target dracoenc_unity
+
+    - name: configure_ios
+      run: cmake . -G Xcode -B build_ios -DCMAKE_BUILD_TYPE=MinSizeRel -DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_ARCHITECTURES=armv7\;armv7s\;arm64 -DCMAKE_OSX_DEPLOYMENT_TARGET=10.0 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF
+    - name: build_ios
+      run: cmake --build build_ios --config MinSizeRel --target dracodec_unity
+
+    - name: package Apple
+      run: |
+        mkdir draco_apple
+        mkdir draco_apple/x86_64
+        mv build_mac/dracodec_unity.bundle draco_apple/x86_64
+        mv build_mac/dracoenc_unity.bundle draco_apple/x86_64
+        mkdir draco_apple/iOS
+        mv build_ios/MinSizeRel-iphoneos/libdracodec_unity.a draco_apple/iOS
+    - name: upload artifact
```

Comment:
Generally speaking I'm fine with everything here accept the automated packaging and release. I think what we'll want to do here, if you're willing, is keep the CI stuff and omit the rest. The additional coverage is great, but the packaging and release automation stuff runs afoul of some of our open source policy.

Reply:
Let me explain why I went the extra mile for this:

When building/updating the libraries for all platforms Unity developers have to copy each single file into the correct location within their project/package. If the structure is already like this in the CI result, this is simplified to a single download and unzip command instead of a lot of manual torque commands.

Fair enough though, If you could confirm that this still is not accepted I'll move it to a fork for personal use. And please feel free to suggest a better solution that would be accepted.